app/controllers/admin/fields_controller.rb

```diff
@@ -103,12 +103,17 @@ def subform
       else
         field_group_id = field[:field_group_id]
         klass = Field.lookup_class(as).classify.constantize
-        klass.new(:field_group_id => field_group_id, :as => as)
+        klass.new(field_group_id: field_group_id, as: as)
       end
 
     respond_with(@field) do |format|
-      format.html { render :partial => 'admin/fields/subform' }
+      format.html { render partial: 'admin/fields/subform' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.